test/e2e/README.md

```diff
@@ -0,0 +1,95 @@
+# E2E Test Instruction
+
+## Steps
+
+1. Deploy Spire
+
+    ```bash
+    helm install docs/demo/deployments/spire/ --generate-name
+    ```
+
+2. Configure Spire
+
+    ```bash
+    ./docs/demo/scripts/spire-config.sh
+    ```
+
+3. Deploy NSM
+
+    ``` bash
+    helm install docs/demo/deployments/nsm-vlan/ --generate-name
+    ```
+
+4. Configure Spire for the testing namespace(meridio to be exist), and trench
+
+    ```bash
+    ns=<testing-namespace>
+    ./docs/demo/scripts/spire.sh default $ns
+    # this step(step 4) needs to be done for multiple times depending on how many trenches are deployed
+    # the <meridio-service-account-name>, for example, for trench-a, is "meridio-trench-a" if helm chart is used, "meridio-sa-trench-a" if operator is used
+    ./docs/demo/scripts/spire.sh <meridio-service-account-name> $ns
+    ```
+
+5. Deploy common resources for the targets
+
+    ```bash
+    helm install examples/target/common/ --generate-name --create-namespace --namespace $ns
+    ```
+
+6. Configure Spire for the targets
+
+    ```bash
+    ./docs/demo/scripts/spire.sh meridio $ns
+    ```
+
+7. Install targets connected to trench-a
+
+    ```bash
+    helm install examples/target/helm/ --generate-name --create-namespace --namespace $ns --set applicationName=target-a --set default.trench.name=<trench-name> --set default.conduit.name=<conduit-name>
```

Comment:
<trench-name> could be set directly to "trench-a" and <conduit-name> we could precise to put load-balancer for helm and lb-fe for operator